### ISSUE TITLE
Use of .pict model : Batch normalization gradient Error

### ISSUE THREAD

**Issue by GU8187** [yining1023/pix2pix_tensorflowjs#5]
Hi everyone,

I'm trying to set up online environment for my new learned pix2pix model. I have exported the .pict model with the guides kindly provided by @yining1023 & Dongphil Yoo.

Here is the error I got at launch. Please check the log below

> tfjs:2 Uncaught (in promise) Error: Batch normalization gradient requires mean and offset to have equal ranks.
> 
>     at f (tfjs:2)
>     at As (tfjs:2)
>     at Object.batchNorm (tfjs:2)
>     at batchnorm (pix2pix.js:34)
>     at pix2pix.js:64
>     at tfjs:2
>     at t.scopedRun (tfjs:2)
>     at t.tidy (tfjs:2)
>     at Object._e [as tidy] (tfjs:2)
>     at Pix2pix.transfer (pix2pix.js:47)
> f @ tfjs:2
> As @ tfjs:2
> batchNorm @ tfjs:2
> batchnorm @ pix2pix.js:34
> (anonymous) @ pix2pix.js:64
> (anonymous) @ tfjs:2
> t.scopedRun @ tfjs:2
> t.tidy @ tfjs:2
> _e @ tfjs:2
> transfer @ pix2pix.js:47
> transfer @ index.js:61
> mouseReleased @ index.js:49
> e._onmouseup @ p5.min.js:9

Concerned url is http://artinabox.fr/p2pampa/

So what can I do? Are there knowledge resources I could investigate, or do you see what parameter(s) I should check and/or modify ?

I'm happy to have trained my own data though : I mean flowers. 
Super best regards to all the community, I hope for your help.

**Comment by GU8187** [comment]
Follow up can be found at [(https://github.com/tensorflow/tfjs/issues/2120)]

**Comment by Dok11** [comment]
Same error here https://yining1023.github.io/pix2pix_tensorflowjs/

**Comment by yining1023** [comment]
Hi @GU8187 and @Dok11 Thank you so much for reporting the issue!

I created a new repo here: https://github.com/yining1023/pix2pix_tensorflowjs_lite. It should be working at https://yining1023.github.io/pix2pix_tensorflowjs_lite/

Please let me know if it works for you.

**Comment by GU8187** [comment]
Yes it does ! Thank you so much @yining1023 for taking care of it and solving the issue.
